pkg/apis/core/types.go

```diff
@@ -335,6 +335,14 @@ type PersistentVolumeSpec struct {
 	// This field influences the scheduling of pods that use this volume.
 	// +optional
 	NodeAffinity *VolumeNodeAffinity
+	// Name of VolumeAttributesClass to which this persistent volume belongs. Empty value
+	// is not allowed. Use nil to indicate that this volume does not belong to any
+	// VolumeAttributesClass. This field is mutable and is changed after a volume has been
+	// updated successfully to a new class.
```

Comment:
Does "and is changed after a volume has been updated successfully to a new class" mean that there is some controller somewhere that changes this automatically, or does is mean that a user should change the value to change the class? (or both?) Can we clarify here in the documentation?